actions.go

```diff
@@ -145,14 +189,21 @@ func (c *Action) EndGroup() {
 
 // SetEnv sets an environment variable.
 func (c *Action) SetEnv(k, v string) {
-	// ::set-env name=<k>::<v>
-	c.IssueCommand(&Command{
-		Name:    setEnvCmd,
-		Message: v,
-		Properties: CommandProperties{
-			"name": k,
-		},
+	err := c.IssueFileCommand(&Command{
+		Name:    envCmd,
+		Message: fmt.Sprintf(envCmdMsgFmt, k, envCmdDelimiter, v, envCmdDelimiter),
 	})
+
+	if err != nil { // use regular command as fallback
```

Comment:
Q: why fallback here? I thought the old way would be deprecated and removed.

Reply:
I picked it up from the TypeScript toolkit. Those guys are also using regular commands as the fallback. I am guessing that it's because the enterprise users may be running an older version, in which case this will prove to be rather helpful.

Reply:
Let's document that then.

Reply:
Will do!